pkg/netpol/connlist/connlist.go

```diff
@@ -409,7 +409,7 @@ func (ca *ConnlistAnalyzer) getConnectionsBetweenPeers(pe *eval.PolicyEngine) ([
 	}
 
 	connsRes := make([]Peer2PeerConnection, 0)
-	peersSet := make(map[string]bool)
+	peers := make([]eval.Peer, 0)
```

Comment:
why not returning `peerList` from line 405?

Reply:
right